lms/djangoapps/discussion/rest_api/utils.py

```diff
@@ -0,0 +1,19 @@
+"""
+Utils for discussion API.
+"""
+
+from datetime import datetime
+
+from pytz import UTC
+
+
+def is_course_discussion_in_blackout(course):
```

Comment:
you don't need this util here. please use existing course util `course.forum_posts_allowed`